std_msgs/String command
---
bool exec_success